README.md

```diff
@@ -128,12 +128,12 @@ In dynamically generated pages (the ones with the device-type and language dropd
 Here is an example:
 
 ```
-Getting started with {{ $device_details.name }} using {{ $language_details.name }}  
+Getting started with {{ $device.name }} using {{ $language.name }}
 
 Select the {{ $device.name }} device type`
 ```
 
-To add additional dynamic properties, see the files in `/config/dictionaries`.
+To add additional dynamic properties, see the files in `/config/dictionaries`. To learn more about dynamically generated pages, see the [README](https://github.com/balena-io/doxx) for the Doxx project, on which the balenaCloud documentation is built.
```

Comment:
```suggestion
To add additional dynamic properties, see the files in `/config/dictionaries`. To learn more about dynamically generated pages, see the [README](https://github.com/balena-io/doxx) for the Doxx project using which the balenaCloud dynamic documentation is generated.
```

Good to refer this, LGTM. Ship it.

Reply:
Thanks. Tweaked your suggestion a little.